modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -1,38 +1,20 @@
-/**
- * {{{description}}}
- * @export
- * @interface {{classname}}
- */
+{{#description}}// {{description}}{{/description}}
 export interface {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{
 {{#additionalPropertiesType}}
     [key: string]: {{{additionalPropertiesType}}}{{#hasVars}} | any{{/hasVars}};
-
 {{/additionalPropertiesType}}
 {{#vars}}
-    /**
-     * {{{description}}}
-     * @type {{=<% %>=}}{<%&datatype%>}<%={{ }}=%>
-     * @memberof {{classname}}
-     */
```

Comment:
See above - i'd prefer this comment over the new one

Reply:
sure thing